src/index.js

```diff
@@ -45,6 +45,9 @@ const filterStack = stackTraceFilter();
 // Wrapper for shadow DOM
 const testDiv = document.createElement('div');
 testDiv.setAttribute('id', 'fcc_test_suite_wrapper');
+// position our user interface at the top layer
+testDiv.style.setProperty('position', 'relative', 'important');
+testDiv.style.setProperty('z-index', '99999', 'important');
```

Comment:
Does it work if these styles are placed in one of the CSS files?

Reply:
@tbushman It doesn't work because of the shadowDOM